examples/c/sim_wt_model_nx6.c

```diff
@@ -336,6 +431,10 @@ int main()
 
 		int acados_return;
 
+		if (nss == 3) // for gnsf: perform precomputation
+			gnsf_precompute(gnsf_dim, in->model, opts, in->T);
```

Comment:
status on embedding this in another function?

Reply:
no solution found yet. It has to be called after the `sim_gnsf_opts_update`. It could be made part of this function, if one adds another input type to the `opts_update` function in the `sim_solver_config`, namely `void *model` or `sim_in *in`.
In my opinion:
`    void (*opts_update) (void *config, void *dims, void *opts, void *model); `
would be a reasonable change.